src/PSDMatrices.jl

```diff
@@ -18,6 +18,16 @@ inv(M::PSDMatrix) = PSDMatrix(inv(M.R'))
 /(B::AbstractVecOrMat, A::PSDMatrix) = B / A.R / A.R'
 copy(M::PSDMatrix) = PSDMatrix(copy(M.R))
 ==(M1::PSDMatrix, M2::PSDMatrix) = M1.R == M2.R  # todo: same as isequal()?!
+# Base.show(io::IO, x::MyString) = print(io, x.s)
+show(io::IO, M::PSDMatrix) = begin
+    print(io, "$(size(M,1))x$(size(M,2)) $(typeof(M)); R=")
+    show(io, M.R)
+end
+show(io::IO, m::MIME"text/plain", M::PSDMatrix) = begin
+    println(io, "$(size(M,1))x$(size(M,2)) $(typeof(M)) ")
+    print(io, " Right square root: ")
+    show(io, m, M.R)
+end
```

Comment:
I dont understand the need for the second function, but I trust you here :) 

Maybe for consistency with the print message above, add the "R="? But feel free to ignore :)
```suggestion
show(io::IO, M::PSDMatrix) = begin
    print(io, "$(size(M,1))x$(size(M,2)) $(typeof(M)); R=")
    show(io, M.R)
end
show(io::IO, m::MIME"text/plain", M::PSDMatrix) = begin
    println(io, "$(size(M,1))x$(size(M,2)) $(typeof(M)) ")
    print(io, " Right square root: R=")
    show(io, m, M.R)
end
```

Reply:
(I am also not sure how standard the show() = begin ... end syntax is, but it seems to work well and I am not the correct person to complain about standard syntax :P)

Reply:
(I am also not sure how standard the show() = begin ... end syntax is, but it seems to work well and I am not the correct person to complain about standard syntax :P)

Reply:
I changed the function definitions to be non-anonymous :)